frontend-react/src/components/Content/PageGenerationTools.test.tsx

```diff
@@ -0,0 +1,30 @@
+import { ContentDirectoryTools } from "./PageGenerationTools";
+
+describe("Page Generation Tools", () => {
+    test("ContentDirectoryTools", () => {
+        /* Testing default instantiation */
```

Comment:
[nit] these three sections could be split out as individual tests in order to give users a better idea of what might be wrong when the tests fail